server/code/setup/cpdb

```diff
@@ -19,8 +19,9 @@
 from __future__ import print_function
 from optparse import OptionParser
 
-import sys
+import glob
```

Comment:
It looks like you're not using glob, actually, since the path doesn't have any globs.

Reply:
Done. I forgot to add the glob to the path.